frontends/web/src/locales/en/app.json

```diff
@@ -1039,28 +1047,15 @@
   },
   "loading": "loading…",
   "manageAccounts": {
-    "addAccount": "Add account",
-    "choose-name": {
-      "nextButton": "Add account",
-      "step": "Name account",
-      "title": "Name your account"
-    },
-    "select-coin": {
-      "nextButton": "Next",
-      "step": "Select coin",
-      "title": "Select cryptocurrency"
-    },
+    "addAccount": "$t(addAccount.title)",
+    "noAccounts": "no accounts found",
     "settings": {
       "hideTokens": "Hide tokens",
       "showTokens": "Show tokens ({{activeTokenCount}})"
     },
-    "success": {
-      "message": "<strong>{{accountName}}</strong> has now been added to your accounts.",
-      "nextButton": "Done",
-      "step": "Finished",
-      "title": "Account added"
-    },
-    "title": "Manage accounts"
+    "title": "Manage accounts",
+    "settingsButton": "$t(manageAccounts.title)",
```

Comment:
delete this and just use the other key in the source code? locize charges also by number of keys, and translators will have to deal with this too.

Reply:
> locize charges also by number of keys

🙈